{"variants":[{"paths":["\/documentation\/sotocore\/awsclienterror\/context"],"traits":[{"interfaceLanguage":"swift"}]}],"kind":"symbol","sections":[],"identifier":{"url":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSClientError\/context","interfaceLanguage":"swift"},"primaryContentSections":[{"declarations":[{"tokens":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"text":"context","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:8SotoCore15AWSErrorContextV","kind":"typeIdentifier","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSErrorContext","text":"AWSErrorContext"},{"kind":"text","text":"?"}],"languages":["swift"],"platforms":["macOS"]}],"kind":"declarations"}],"schemaVersion":{"major":0,"minor":3,"patch":0},"hierarchy":{"paths":[["doc:\/\/codes.soto.soto-core\/documentation\/SotoCore","doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSClientError"]]},"abstract":[{"text":"additional context information related to the error","type":"text"}],"metadata":{"symbolKind":"property","fragments":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"text":"context","kind":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"AWSErrorContext","preciseIdentifier":"s:8SotoCore15AWSErrorContextV"},{"kind":"text","text":"?"}],"title":"context","role":"symbol","externalID":"s:8SotoCore14AWSClientErrorV7contextAA15AWSErrorContextVSgvp","modules":[{"name":"SotoCore"}],"roleHeading":"Instance Property"},"references":{"doc://codes.soto.soto-core/documentation/SotoCore/AWSErrorContext":{"type":"topic","url":"\/documentation\/sotocore\/awserrorcontext","title":"AWSErrorContext","kind":"symbol","role":"symbol","navigatorTitle":[{"kind":"identifier","text":"AWSErrorContext"}],"fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"text":"AWSErrorContext","kind":"identifier"}],"identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSErrorContext","abstract":[{"type":"text","text":"Additional information about error"}]},"doc://codes.soto.soto-core/documentation/SotoCore/AWSClientError":{"kind":"symbol","abstract":[],"type":"topic","title":"AWSClientError","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSClientError","fragments":[{"text":"struct","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"AWSClientError"}],"url":"\/documentation\/sotocore\/awsclienterror","role":"symbol","navigatorTitle":[{"kind":"identifier","text":"AWSClientError"}]},"doc://codes.soto.soto-core/documentation/SotoCore/AWSClientError/context":{"fragments":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"text":"context","kind":"identifier"},{"kind":"text","text":": "},{"preciseIdentifier":"s:8SotoCore15AWSErrorContextV","kind":"typeIdentifier","text":"AWSErrorContext"},{"kind":"text","text":"?"}],"role":"symbol","type":"topic","title":"context","kind":"symbol","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSClientError\/context","url":"\/documentation\/sotocore\/awsclienterror\/context","abstract":[{"text":"additional context information related to the error","type":"text"}]},"doc://codes.soto.soto-core/documentation/SotoCore":{"type":"topic","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore","kind":"symbol","url":"\/documentation\/sotocore","abstract":[{"type":"text","text":"The core framework for Soto, a Swift SDK for AWS"}],"role":"collection","title":"SotoCore"}}}